### ISSUE TITLE
Unnecessary Call to Softmax

### ISSUE THREAD

**Issue by nahuakang** [aymericdamien/TensorFlow-Examples#176]
https://github.com/aymericdamien/TensorFlow-Examples/blob/72707e2314f38217f9977565982ea22e99830517/examples/3_NeuralNetworks/convolutional_network_raw.py#L100

Hi Aymeric, 

first of all thanks a lot for building up this repository. I've been using it to review TensorFlow and what I've learned on deep learning. 

I'm wondering if this call of `prediction = tf.nn.softmax(logits)` is necessary since you are calling `tf.nn.softmax_cross_entropy_with_logits` later? The [documentation](https://www.tensorflow.org/api_docs/python/tf/nn/softmax_cross_entropy_with_logits) states that:
> WARNING: This op expects unscaled logits, since it performs a softmax on logits internally for
> efficiency. Do not call this op with the output of softmax, as it will produce incorrect results.

**Comment by nahuakang** [comment]
Apologies, I realized the `prediction` call is used for evaluation of the model. Please close this issue.
